src/js/components/awardv2/idv/InfoTooltip.jsx

```diff
@@ -51,15 +51,22 @@ export default class InfoTooltip extends React.Component {
 
     closeTooltip() {
         this.setState({
-            showInfoTooltip: false
+            showInfoTooltip: true
         });
     }
 
     measureOffset() {
         const targetElement = this.referenceDiv;
         const offsetTop = targetElement.offsetTop - 15;
         let tooltipWidth = 375;
-        if (this.props.wide) {
+        // is the t.t. in a section that takes up the full width of the screen?
+        const isTooltipJustifiedRight = (window.innerWidth - targetElement.offsetLeft) < window.innerWidth / 6;
```

Comment:
Not sure I'm following the logic here - why is offset compared to `window.innerWidth / 6`?

Reply:
@ebdabbs, 

The scenario I'm seeking to recognize is:

- Tooltip is on a full-width component
- Tooltip is invoked w/ the prop `wide`

In this case, the pixel value of `offsetLeft` for the tooltip is very near the value of the `window.innerWidth`. On my laptop, `window.innerWidth - offsetLeft` is ~90px whereas on my monitor it's about 240px. In either case, it's approximately less than `1/6` of `window.innerWidth`.

So, to answer your question, it's somewhat arbitrary, but this value was reliable during my testing. During my testing, I was able to confirm that `window.innerWidth - offsetLeft` is larger than `1/6` of the screen unless the conditions of this scenario, stated above, apply.

That said, my only real goal here was to not use a fixed width, given the variety of screen sizes that will affect the value of `window.innerWidth`.